api2/pkg/webapp/casetypes.go

```diff
@@ -41,6 +41,49 @@ func (h *Handler) CaseTypes(w http.ResponseWriter, req *http.Request) {
 	}
 }
 
+func (h *Handler) PostCaseType(
```

Comment:
@senyaoh I had to move this function above the others and fix the missing } problem!